decidim-accountability/spec/types/integration_schema_spec.rb

```diff
@@ -75,16 +80,14 @@
           updatedAt
           userAllowedToComment
           weight
+          url
+          proposals { id }
         }
       }
     )
     end
   end
   let(:component_type) { "Accountability" }
-  let!(:current_component) { create(:accountability_component, participatory_space: participatory_process) }
-  let!(:result) { create(:result, component: current_component, taxonomies:) }
-  let!(:timeline_entry) { create(:timeline_entry, result:) }
-
   let(:accountability_single_result) do
```

Comment:
same here 

I see taxonomies is out of order, but I understand because we were changing from `categories`. Lets fix it